app/views/RoomInfoEditView/index.js

```diff
@@ -440,230 +440,238 @@ class RoomInfoEditView extends React.Component {
-							theme={theme}
-							testID='room-info-edit-view-description'
-						/>
-						<RCTextInput
-							inputRef={(e) => { this.topic = e; }}
-							label={I18n.t('Topic')}
-							value={topic}
-							onChangeText={value => this.setState({ topic: value })}
-							onSubmitEditing={() => { this.announcement.focus(); }}
-							theme={theme}
-							testID='room-info-edit-view-topic'
-						/>
-						<RCTextInput
-							inputRef={(e) => { this.announcement = e; }}
-							label={I18n.t('Announcement')}
-							value={announcement}
-							onChangeText={value => this.setState({ announcement: value })}
-							onSubmitEditing={() => { this.joinCode.focus(); }}
-							theme={theme}
-							testID='room-info-edit-view-announcement'
-						/>
-						<RCTextInput
-							inputRef={(e) => { this.joinCode = e; }}
-							label={I18n.t('Password')}
-							value={joinCode}
-							onChangeText={value => this.setState({ joinCode: value })}
-							onSubmitEditing={this.submit}
-							secureTextEntry
-							theme={theme}
-							testID='room-info-edit-view-password'
-						/>
-						<SwitchContainer
-							value={t}
-							leftLabelPrimary={I18n.t('Public')}
-							leftLabelSecondary={I18n.t('Everyone_can_access_this_channel')}
-							rightLabelPrimary={I18n.t('Private')}
-							rightLabelSecondary={I18n.t('Just_invited_people_can_access_this_channel')}
-							onValueChange={this.toggleRoomType}
-							theme={theme}
-							testID='room-info-edit-view-t'
-						/>
-						<SwitchContainer
-							value={ro}
-							leftLabelPrimary={I18n.t('Collaborative')}
-							leftLabelSecondary={I18n.t('All_users_in_the_channel_can_write_new_messages')}
-							rightLabelPrimary={I18n.t('Read_Only')}
-							rightLabelSecondary={I18n.t('Only_authorized_users_can_write_new_messages')}
-							onValueChange={this.toggleReadOnly}
-							disabled={!permissions[PERMISSION_SET_READONLY] || room.broadcast}
-							theme={theme}
-							testID='room-info-edit-view-ro'
-						/>
-						{ro && !room.broadcast
-							? (
-								<SwitchContainer
-									value={reactWhenReadOnly}
-									leftLabelPrimary={I18n.t('No_Reactions')}
-									leftLabelSecondary={I18n.t('Reactions_are_disabled')}
-									rightLabelPrimary={I18n.t('Allow_Reactions')}
-									rightLabelSecondary={I18n.t('Reactions_are_enabled')}
-									onValueChange={this.toggleReactions}
-									disabled={!permissions[PERMISSION_SET_REACT_WHEN_READONLY]}
-									theme={theme}
-									testID='room-info-edit-view-react-when-ro'
-								/>
-							)
-							: null
-						}
-						{room.broadcast
-							? [
-								<Text style={styles.broadcast}>{I18n.t('Broadcast_Channel')}</Text>,
-								<View style={[styles.divider, { borderColor: themes[theme].separatorColor }]} />
-							]
-							: null
-						}
-						{!compareServerVersion(serverVersion, '3.0.0', methods.lowerThan) ? (
+						<TouchableOpacity activeOpacity={1} onPress={() => Keyboard.dismiss()}>
```

Comment:
It doesn't look correct.
We shouldn't add a button with the size of the scroll just to fix a keyboard dismiss.